resources/lang/en.js

```diff
@@ -1,5 +1,7 @@
 /* exported L10N */
 const L10N = {
+    'collage_cntdwn_time': 'Set collage-countdown timer',

```

Comment:
Is "Set collage-countdown" not sufficient?

Reply:
I think one of them is because it's an input and not selection,  will have to check on the running system which one is used. I think there's one more I can remove which I'll do while on it.